exif.cc

```diff
@@ -1251,9 +1208,9 @@ exif_write_apps()
         exif_put_long(IFD1, IFD1_TAG_JPEG_OFFS, 0, len);
       }
 
-      QUEUE_FOR_EACH(&app->ifds, e1, t1) {
-        exif_ifd_t* ifd = reinterpret_cast<exif_ifd_t *>(e1);
-        sortqueue(&ifd->tags, exif_sort_tags_cb);
+      for (auto& ifd_instance : app->ifds) {
```

Comment:
Could this loop be combined with the one above or does GPS_IFD interfere with the sort? Thinking about readability here(if that's not a lost cause in this module :-) not the performance const of repeating it.)

Reply:
maybe, but it will have to wait.